aten/src/ATen/native/native_functions.yaml

```diff
@@ -1245,6 +1248,18 @@
 - func: conv3d(Tensor input, Tensor weight, Tensor? bias=None, int[3] stride=1, int[3] padding=0, int[3] dilation=1, int groups=1) -> Tensor
   use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
 
+- func: conv1d.padding_mode(Tensor input, Tensor weight, Tensor? bias=None, int[1] stride=1, str padding="valid", int[1] dilation=1, int groups=1) -> Tensor
```

Comment:
As mentioned in some other comments, the "padding_mode" terminology here is a bit confusing since "padding mode" is used in Conv1d / Conv2d / Conv3d modules to describe the padding fill values (e.g. zeros, reflect, etc.).

Maybe this can be changed to just `padding`?